src/components/ProgressBar/ProgressBar.js

```diff
@@ -1,48 +1,54 @@
 import React from 'react';
-import './ProgressBar.scss'
+import styled from 'styled-components';
 
-const ProgressBar = ({ item }) => {
-    // ******Please use StyledComponent
-    const containerStyles = {
-        height:42,
-        width:'100%',
-        marginTop:10,
-        marginBottom:12,
-        textAlign:'center',
-    }
 
-    const fillerStyles = {
-        height:'100%',
-        width: `${item.completed > 100 ? 100 : item.completed}%`,
-        backgroundColor: `${item.completed > 100 ? 'red' : 'lightblue'}`,
-        textAlign: 'center',
-    }
+const ContainerStyles = styled.div`
+        height: 42px;
+        width: 100%;
+        margin: 0 auto;
+        position: relative;
+        box-sizing: border-box;
+        border: .1em solid rgb(199, 199, 199);
+    `;
 
-    const lableStyles = {
-        color:'black',
-        textAlign:'center',
-    }
+const FillerStyles = styled.div`
+        height: 100%;
+        width: ${props => props.completed > 100 ? '100%' : props.completed +'%' };
+        background-color: ${props => props.completed > 100 ? 'red' : 'lightblue'};
+        text-align: center;
+        z-index: -1;
+        transition: width 1s;
+    `;
 
-    return (
-        <div
-            className='bar_container'
-            style={containerStyles}
+const LableStyles = styled.div`
+        color:black;
```

Comment:
add space after :
other places need to be fixed as well.

Reply:
I agree.